CodeEdit/Features/Settings/SettingsView.swift

```diff
@@ -191,11 +205,46 @@ struct SettingsView: View {
         .environmentObject(model)
         .onAppear {
             selectedPage = Self.pages[0].page
```

Comment:
Quick note, this code looks like it's trying to set the settings page to 'General' whenever the settings window appears. Specifically, this only ever happens once, when the settings window first appears. When the window is closed and then reopened, the last settings menu the user selected is still selected. It is not reset to `Self.pages[0].page`. Is this intended? Or is this code trying to reset to the 'General' settings when the window is closed? If not, I can remove this line since it's redundant.

Reply:
Each time the settings window is open, it should open to general.

Reply:
I'll remove this line from the code since it's not actually doing anything. In Xcode, the settings menu will always be where the user last selected though. Let me see if I can fix this functionality in this PR if it's simple enough, otherwise I'll make a new issue.